shared_model/validators/transaction_validator.hpp

```diff
@@ -316,8 +332,11 @@ namespace shared_model {
           : field_validator_(field_validator) {}
 
      public:
-      explicit TransactionValidator(std::shared_ptr<ValidatorsConfig> config)
-          : TransactionValidator(FieldValidator{config}) {}
+      explicit TransactionValidator(
+          const std::shared_ptr<ValidatorsConfig> &config)
+          : TransactionValidator(FieldValidator{config}) {
```

Comment:
```suggestion
          : TransactionValidator(FieldValidator{config}), validators_config_{config} {
```

Reply:
It is impossible with constructor delegation because delegation is possible only if the object is fully initialization